src/Microsoft.AspNetCore.Identity.EntityFrameworkCore/Resources.resx

```diff
@@ -117,6 +117,14 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="NotIdentityRole" xml:space="preserve">
+    <value>AddEntityFrameworkStores can only be called with a role that derives from IdentityRole&lt;TKey&gt;. If you are specifying more generic arguments, use IdentityBuilder.AddRoleStore&lt;TStore&gt;() instead.</value>
```

Comment:
Couple of observations:
- I don't think the mention of `IdentityBuilder` helps here. I would just remove it. 
- Customers also need to derive their own store classes (by overriding a method in the abstract base class), right? If that is the case, can you also mention it in the messages?